qoffer-infrastructure/src/main/groovy/life/qbic/portal/qoffer2/web/views/create/offer/SelectItemsView.groovy

```diff
@@ -198,6 +195,9 @@ class SelectItemsView extends VerticalLayout{
         VerticalLayout projectManagementLayout = new VerticalLayout(projectManagementGrid, quantityManagement)
         projectManagementLayout.setSizeFull()
 
+        HorizontalLayout overview = new HorizontalLayout(overviewGrid)
```

Comment:
What's happening here? And should this be part of this PR?

Reply:
It was a leftover of me trying to fix the missing grid :D good catch!